tools/clang/unittests/HLSLExec/LongVectors.tpp

```diff
@@ -513,30 +503,73 @@ DataTypeT LongVector::TestConfig<DataTypeT, LongVectorOpTypeT>::ComputeExpectedV
   case LongVector::BinaryOpType_ScalarMax:
     return (std::max)(A, B);
   default:
-    LOG_ERROR_FMT_THROW(L"Unknown BinaryOpType: %d", OpTypeTraits.OpType);
+    LOG_ERROR_FMT_THROW(L"Unknown BinaryOpType: %d", opTypeTraits.OpType);
     return DataTypeT();
   }
 }
 
 template <typename DataTypeT, typename LongVectorOpTypeT>
-DataTypeT LongVector::TestConfig<DataTypeT, LongVectorOpTypeT>::ComputeExpectedValue(const DataTypeT &A, const DataTypeT &B) const {
-  if(!IsBinaryOp())
+DataTypeT LongVector::TestConfig<DataTypeT, LongVectorOpTypeT>::computeExpectedValue(const DataTypeT &A, const DataTypeT &B) const {
+  if(!isBinaryOp())
     LOG_ERROR_FMT_THROW(
-        L"ComputeExpectedValue(const DataTypeT &A, const DataTypeT &B) called "
+        L"computeExpectedValue(const DataTypeT &A, const DataTypeT &B) called "
         L"on a unary op: %d",
-        OpTypeTraits.OpType);
+        opTypeTraits.OpType);
+
+  return computeExpectedValue(A, B, static_cast<LongVector::BinaryOpType>(opTypeTraits.OpType));
+}
+
+
+template <typename DataTypeT, typename LongVectorOpTypeT>
```

Comment:
Is there a reason this has been moved up from where it was originally?

Reply:
I think at the time it made sense to me to move it to group it with the binary op version. Not sure it really matters though.